source/guides/network/creating-a-private-test-net.rst

```diff
@@ -5,165 +5,232 @@
-    │ Address     │ SCVG35-ZSPMYP-L2POZQ-JGSVEG-RYOJ3V-BNIU3U-N2E6                   │
-    ├─────────────┼──────────────────────────────────────────────────────────────────┤
-    │ Public Key  │ 654...321                                                        │
-    ├─────────────┼──────────────────────────────────────────────────────────────────┤
-    │ Private Key │ 123...456                                                        │
-    └─────────────┴──────────────────────────────────────────────────────────────────┘
-
-As the name suggests, the **private key has to be kept secret at all times**.
-Anyone with access to the private key ultimately has control over the account.
-On the other hand, you can share securely the public and address of your account with other participants of the network to receive transactions from them.
+    symbol-bootstrap start -p <preset> -a <assembly>
+
+That's right, a test network or node can be instantiated and booted with a single command!
+
+*********************************************
+Check that the REST gateway is up and running
+*********************************************
+
+If the selected configuration includes an API node, you can verify that the node is running by opening a new browser tab and going to: ``localhost:3000/chain/height``.
+
+****************************
+Retrieving the node accounts
+****************************
+
+Depending of the selected configuration, one or more nodes with corresponding accounts have been created. To interact with any of these accounts (to transfer mosaics **to** it, for example) you need its address or its public key. To fully control an account (to transfer mosaics **from** it, for example) you need its private key.
+
+All this information can be retrieved from a file in the ``config`` folder:
+
+``target/config/generated-addresses/addresses.yml``
+
+As an example:
+
+.. code-block:: yaml
+
+    networkType: 152
+    nemesisGenerationHashSeed: 7BFC536990108CA923B2715DE6B8E47E6BB56C945293BF4FC22C5AF895F61E62
+    nodes:
+        -
+            signing:
+                # Keys for the account of the peer-node-0
+                privateKey: 6D0C9F945FA2AED90CF072A8BB2898C40A9D3C53B1E8E8541D3206673EF7D3DC
+                publicKey: C2BD21E4F9261247A4CBE75DA8683978E0F1FFF34AAB17BEBC21E7B9E0E17A9F
+                address: TAMEGYVY6GVGXCLBIEH72XU4D2OSTH2MIOOY4QQ
+            vrf:
+                privateKey: 8F02E4D7A9C3DD946561AF1D19A746443C46A0C4888EA61AAF01A2455043EB3E
+                publicKey: D67601AC6767F8A3C97FFDB0D9D737A943FFFA6E69C2C6527B0ED32A4E41B443
+                address: TCR6ZWX3UP3TKJK3BSE6ARJ4WAF3KHFFUWJ7EWI
+            ssl:
+                privateKey: 76e9f535f1bd15d629e5dc621c9a56e7dcb52c72de74ffed44eb8f7ae8295cbc
+                publicKey: 3d68cdaa4e48a14bd875d4ca18e044522be5f602e8c4f37c1c65eafdfaa19110
+            type: peer-node
+            name: peer-node-0
+            friendlyName: peer-node-0
```

Comment:
I'll add the `mosaic:` section that shows the nemesis accounts that the dev can use as either test accounts or use them to send funds to new test accounts.

Reply:
OK, but you will need to explain me what they are :)

Reply:
Example in the addresses.yml:

```
mosaics:
    currency:
        -
            privateKey: 18FF87F65F76B74A031A6E32BAE4D97D1BC200E2F22176FE6C18847E24DA87B0
            publicKey: CA1B1F687D7113831FDAA972D96466BACEF627C79FE6442B5635851409866B64
            address: TC6A25QFDR675Y47OAGVGEDS6XMH4YDSZ4TIZ4A
        -
            privateKey: ABE88A8798B5947FBCE549264D087D3F7407D3AED634E7511C79D5C823B1E2E8
            publicKey: DFC03745BF98C0B7D642E9B4EB2579181A48A979C9B78C8125690C13C7AAB290
            address: TAAEPLRS2M6K42HTCCHOJM4UIVMPW6JOPZSIWVY
        -
            privateKey: A54BB2880C94F60F914F94C0231954564333C1ADB0F5009B0DB860276F1307DC
            publicKey: 428DA7CD8469E5F822CA549E6A82FC8E4208096E683C90B5DD58861294F7B049
            address: TD3PIYQ6PZPV5PO6YJGP4TH2MO6DYMIX5GUQQEY
        -
            privateKey: E37C7AFDA94F8216447A38AE96BFB8262279BE6DCB878ECFA47E4AA394389B89
            publicKey: CB330E700CA65165CCC227F5827EE567816B305F1ACC350DF1A5159B8BDE7E7D
            address: TCOWFTD6A3US57QI4I5FSFRAHSA6MTFUD3LXUHY
        -
            privateKey: 02E98DDF68D7BCA34718C61B9EA46378EEC3903E2CC1086F53DD410478D99CEC
            publicKey: BB2B046267876657D00936029F0F013167607998E34E92F3F261479C3F995A94
            address: TCEBJDTMKOP7VQBUMPVW2NPYU3LA4TMSEITGRBY
    harvest:
        -
            privateKey: 963831567EB504BD1D45CDDD59DA0702DB4717EAA938A1F7F3D19D5347E22346
            publicKey: A058D7EFA58E1D88B1AEFE1E7698D3D70E2AF61116ED0AC88DAAF65217C2C46A
            address: TALSYSIUPGDYIHTI3QGLDNLCQT36ZXTCKTK7XPI
        -
            privateKey: EB6AA0690E7C4AE8AD03F078E0DFBA561FBEE74CAB407DB2D5EACF789FCA370B
            publicKey: 2530CDFCF41B8763C977AD215D995B65C9A846D731E53C73DFD0D4AF3F1F0120
            address: TBE6KGCPO6D2O3JIV5TLBFI3X6VMS6EV5LU6W3Q
```
Those are generated accounts that hold the main currency and harvest currency when booting a new network. Users can use those for testing (directly or by transfer funds from these to new testing accounts). I would use those accounts instead of the nodes accounts for regular testing use. I would use node accounts if you want to do node-specific actions like voting key link, vrf, or to send harvesting funds